### ISSUE TITLE
Clipboard crash

### ISSUE THREAD

**Issue by Seilican** [ldtteam/minecolonies#10267]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Are you using the latest MineColonies Version?

- [X] I am running the latest **beta/release** version of MineColonies for my Minecraft version.
I am also running the latest versions of other mods that are part of my problem.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### Minecraft Version

1.20

### MineColonies Version

atm 9

### Structurize Version

atm 9

### Related Mods and their Versions

atm 9

### Reproduction Steps

have 2 colonies and open clipboard crash

### Logs

https://pastebin.com/XaKtRa0h

### Anything else?

why have it hard to give you info to help no i have not looked any where i am sharing crash log to help you know its going on

### Footer

---
## Viewers
* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Talyda** [comment]
Update minecolonies and dependencies

**Comment by Mistsiege** [comment]
Just had this issue happen on the Raid Competition Server, https://pastebin.com/mU6WCXbR

Appears to be happening after a few looks at the clipboard, but it's crashed me a few times in the past half hour.  I believe it may have been triggered when I saw nothing on the board despite 23 notifications on the icon for it, I reassigned it to the colony and started having the issue.  May be unrelated, but it's the only thing I can think of troubleshooting-wise.
